src/components/GardensDashboard.js

```diff
@@ -1,27 +1,122 @@
-import React from 'react'
-import { GU, Link } from '@1hive/1hive-ui'
+import React, { useCallback } from 'react'
+import { GU, shortenAddress, textStyle, useTheme } from '@1hive/1hive-ui'
 import { useGardens } from '../providers/Gardens'
+import { useHistory } from 'react-router'
+
+import defaultGardenSvg from '../assets/defaultGarden.svg'
+import defaultTokenSvg from '../assets/defaultToken.svg'
 
 function GardensDashboard() {
   const { gardens } = useGardens()
+
   return (
     <div
       css={`
         padding: ${3 * GU}px;
         display: grid;
-        grid-template-columns: repeat(3, 1fr);
+        grid-gap: ${2 * GU}px;
+        grid-template-columns: repeat(4, 1fr);
       `}
     >
       {gardens.map((garden, index) => (
-        <div key={index}>
-          <Link href={`#/garden/${garden.address}`} external={false}>
-            {garden.id}
-          </Link>
-          <span>{garden.proposalCount}</span>
-        </div>
+        <GardenCard key={garden.id} garden={garden} />
       ))}
     </div>
   )
 }
 
+function GardenCard({ garden }) {
+  const theme = useTheme()
+  const history = useHistory()
+  const handleSelectGarden = useCallback(() => {
```

Comment:
not sure if we want to keep the buildGardenPath or not but if we decide to use it we should use it here also right?

Reply:
Not in this case because it assumes the current location is a garden